xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/util/TenantUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.icthh.xm.commons.domain.event.util;
+
+import com.icthh.xm.commons.tenant.TenantContextUtils;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class TenantUtil {
```

Comment:
Seems it is better to add `normalizeTenant` directly to TenantContextUtils instead of creating one more utility class.